web3rpc/sdk/client/java/web3j-ext-test-wrapper/build.gradle

```diff
@@ -15,7 +15,7 @@ repositories {
 }
 
 dependencies {
-    api "foundation.klaytn:web3rpc-java:v0.9.0"
+    api "foundation.klaytn:web3rpc-java:v1.0.1"
     implementation "org.web3j:core:4.9.7"
```

Comment:
The generated sdk have version 1.0.1 so i need to change build.gradle version of web3rpc-java to match the local build